internal/bminventory/inventory.go

```diff
@@ -848,7 +833,24 @@ func (b *bareMetalInventory) V2ImportClusterInternal(ctx context.Context, kubeKe
 		KubeKeyNamespace: kubeKey.Namespace,
 	}
 
-	err = validations.ValidateClusterNameFormat(clusterName)
+	// Day2 supports only x86_64 for now
+	if inputOpenshiftVersion != "" {
```

Comment:
I'm not sure why we need to bother about all this?
What code in the add host flow require an openshift release image?

Reply:
Is it just for validating the assisted-service "knows" this release?
it makes sense for the day1 cluster where the OCP release is being used but in day 2 I don't think the assisted service has any use for it so I think the assisted service can be version agnostic.